test_files/export.in.ts

```diff
@@ -0,0 +1,2 @@
+export * from './export_helper';
```

Comment:
It might be too much to ask, but from experience I find that another level of reexport might have issues where a single level doesn't. That is adding `export * from './export_helper_2';` in `export_helper`.